docs/settings.rst

```diff
@@ -49,6 +49,19 @@ All items in the list of ``sphinx-needs.activateFiles`` must match the VsCode su
 
    * After update settings of ``activateFiles``, restart VsCode.
 
+.. _explorerOptions:
+
+``sphinx-needs.explorerOptions``
+--------------------------------
+
+Specify **a list of comma seperated string of need options** to display at :ref:`sphinxNeedsExplorer`. Default is: [].
```

Comment:
I think we should have a default here, otherwise it is a little bit confusing that the user can "open/collpase" the needrow, but no entries are shown.

By default it should be: status, links, tags.